README.md

```diff
@@ -0,0 +1,82 @@
+# Kartenforum Georeference
+
+This repository contains the code for deploying a georeference service as well as a daemon for an asynchrone permanent update of the georeference data basis.
+
+## Install
+
+The tools are developed with Python 3.9.7. They rely on the following dependencies:
+
+```
+apt-get install python3 python3-virtualenv libpq-dev gdal-bin libgdal-dev gcc uwsgi
+```
+
+The python dependencies are installed within a virtual environment. Therefor run the following commands:
+
+```
+virtualenv python_env
+./python_env/bin/pip install -e ".[testing]"
```

Comment:
Unfortunately, I fail in this step with the following error. Do you have any idea, what I'm missing?

I haven't the same environment as you. My system ist a Manjaro-Linux.

```
Collecting GDAL==3.2.2
  Using cached GDAL-3.2.2.tar.gz (601 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      /home/ab/projects/kartenforum_georeference/python_env/lib/python3.10/site-packages/setuptools/_distutils/dist.py:275: UserWarning: Unknown distribution option: 'use_2to3_fixers'
        warnings.warn(msg)
      /home/ab/projects/kartenforum_georeference/python_env/lib/python3.10/site-packages/setuptools/_distutils/dist.py:275: UserWarning: Unknown distribution option: 'use_2to3_exclude_fixers'
        warnings.warn(msg)
      error in GDAL setup command: use_2to3 is invalid.
      WARNING: numpy not available!  Array support will not be enabled
      [end of output]
```

Reply:
Arch based distros use a more current version of the python packages per default. 

In my situation it helped to install an older version of the setuptools within the virtualenvironment:

`./python_env/bin/pip install setuptools=57.5.0/`